src/rider/main/kotlin/me/seclerp/rider/plugins/efcore/cli/execution/NotificationCommandResultProcessor.kt

```diff
@@ -23,19 +24,19 @@ public class NotificationCommandResultProcessor(
                 .notify(project)
         } else {
             val errorTextBuilder = StringBuilder()
-            errorTextBuilder.append("Command: ${result.command}")
+            errorTextBuilder.append("${EfCoreUiBundle.message("command")} ${result.command}")
```

Comment:
_(Minor)_ I recommend updating the message to use a pattern and then this:
```suggestion
            errorTextBuilder.append(EfCoreUiBundle.message("command", result.command))
```